main.py

```diff
@@ -1012,6 +1018,24 @@ def get(self, user):
         url = self.gen_photo_url(args[1], args[0], 'school_photos')
         self.redirect(url)
 
+class SearchHandler(BaseHandler):
+    def get_url_prefix(self, grade):
+        if grade:
+            return "student"
+        else:
+            return "teacher"
+
+    def get(self, keyword):
+        results = []
+        for schedule in SCHEDULE_INFO:
```

Comment:
do you want to sort this in any way? Right now it looks like it returns a potentially very large # of results, which you probably want to cap or sort somehow